srcpkgs/apcupsd/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'apcupsd'
+pkgname=apcupsd
+version=3.14.14
+revision=1
+build_style=gnu-configure
+configure_args="
+ --disable-install-distdir
+ --enable-modbus-usb
+ --enable-usb
+ --sysconfdir=/etc/apcupsd
+ --with-dev=
+ --with-halpolicydir=
+ --with-pwrfail-dir=/etc/apcupsd
+ --with-upscable=usb
+ --with-upstype=usb"
+hostmakedepends="autoconf automake pkg-config"
+makedepends="libusb-compat-devel net-snmp-devel"
+depends="libusb-compat net-snmp nullmailer"
+conf_files="/etc/apcupsd/*"
+short_desc="APC UPS daemon with integrated tcp/ip remote shutdown"
+maintainer="Bas Ploeger <s.c.w.ploeger@gmail.com>"
+license="GPL-2.0-only"
+homepage="http://www.apcupsd.org/"
+distfiles="${SOURCEFORGE_SITE}/apcupsd/apcupsd-${version}.tar.gz"
+checksum="db7748559b6b4c3784f9856561ef6ac6199ef7bd019b3edcd7e0a647bf8f9867"
+
+# Avoid passing an invalid arg 'strip' to the install command for the binaries.
+make_install_args="STRIP="
```

Comment:
```suggestion
```

that message in the logs can just be ignored, it's just xbps-src's strip wrapper saying it's being called. It gets stripped properly in a later build phase.

Reply:
This line is needed to work around a problem in the package's configure/make setup. If I omit it, then `./xbps-src install apcupsd` fails with:

```
  COPY  apcupsd => /destdir//apcupsd-3.14.14/usr/bin/apcupsd
/usr/bin/install: target '/destdir//apcupsd-3.14.14/usr/bin/apcupsd' is not a directory
make[2]: *** [Makefile:49: install-base] Error 1
make[1]: *** [../autoconf/targets.mak:69: install] Error 2
make: *** [autoconf/targets.mak:111: src_DIR] Error 2
=> ERROR: apcupsd-3.14.14_1: do_install: '${make_cmd} DESTDIR=${DESTDIR} ${make_install_args} ${make_install_target}' exited with 2
=> ERROR:   in do_install() at common/build-style/gnu-configure.sh:40
```

The problem is that the `install` command that `make` is executing, gets an argument `strip` rather than `--strip` (I am not sure why, but this seems an error in the configure/make files). This wrong argument means `install` tries to copy files `strip` and `apcupsd` into a _directory_ `/usr/bin/apcupsd`, which is not the intention. The intention is to copy file `apcupsd` to file `/usr/bin/apcupsd`. 

The only way I found to work around this error, is by clearing the env var `STRIP`, which ensures the `strip` argument is not passed to `install`.